samples/video/mt9m114/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _mt9m114:
+
+MT9M114: CMOS image sensor
+########################################
+
+Description
+***********
+
+This sample application simply retrieves frames from MT9M114 sensor via video
+API.
+
+References
+**********
+
+ - MT9M114: https://www.onsemi.com/PowerSolutions/product.do?id=MT9M114
+
+Prerequisite
+************
+
+This sample requires MT9M114 to be connected.
```

Comment:
connected to what?  Need to include a board reference and example of building the sample (see most any other sample for an example of using the ``.. zephyr-app-commands`` directive to document a build example).

Also looks like a serial connection is needed (to see the sample output).

Reply:
done